editor/selectors.js

```diff
@@ -3,6 +3,7 @@
  */
 import moment from 'moment';
 import { first, last, get } from 'lodash';
+import { createSelector } from 'reselect';
```

Comment:
mmm! Good point about the selectors with arguments, I think we'll need this quickly. Though according to their docs, we could write this

```js
const createSelectorWithArguments = ( extractors, resolver ) => {
	const memoizedResolver = memoize( resolver );
	const memoizedSelector = createSelector( extractors, memoizedResolver );
	return ( state, ...args ) => {
		const selector = memoizedSelector( state );
		return selector( ...args );
	};
};
```